src/components/metric-chart.component.ts

```diff
@@ -59,14 +64,25 @@ export class MetricChartComponent implements OnChanges {
   @Input() timestampLabel = 'Timestamp';
   @Input() rawData?: NumericDataPoint[];
   @Input() statsData?: NumericBucketPoint[];
+  @Input() multiData: IMultiDataPoint[];
   @Input() forecastDataPoints: IPredictiveMetric[];
-  @Input() showDataPoints = false;
+  @Input() showDataPoints = true;
   @Input() previousRangeData = [];
   @Input() annotationData = [];
   @Input() yAxisUnits: string;
-  @Input() multiData: IMultiDataPoint[];
   @Input() raw: false;
   @Input() buckets = 60;
+  @Output() timeRangeChange = new EventEmitter();
+  timeRangeValue: TimeRange = 43200;
+  @Input()
```

Comment:
Is this correct `Input()` on get function? I guess you wanted to put `Output()` to recieve `timeRangevalue`.